scripts/docgen/docgen.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /*  
   +----------------------------------------------------------------------+
-  | PHP Version 5                                                        |
+  | PHP Version 7                                                        |
```

Comment:
We've removed this in the core, if you want to drop it here as well: https://github.com/php/php-src/pull/4732

Reply:
Yep, doesn't make much sense to have it here, but I assume there are several other files where it could be removed; might better be done as separate commit.

Reply:
Thanks, done.